discovery-infra/node_controlers/node_controler.py

```diff
@@ -0,0 +1,132 @@
+class NodeControler(object):
+
+    def __init__(self):
```

Comment:
Remove empty `__init__` and `__del__` and keep the default ctor&dtor.

Reply:
done